python/pacemaker/buildoptions.py.in

```diff
@@ -0,0 +1,53 @@
+# Copyright 2023 the Pacemaker project contributors
+#
+# The version control history for this file may have further details.
+#
+# This source code is licensed under the GNU Lesser General Public License
+# version 2.1 or later (LGPLv2.1+) WITHOUT ANY WARRANTY.
+
+__all__ = ["BuildOptions"]
+
+class BuildOptions(object):
+    # Path to the bash shell
+    BASH_PATH = "@BASH_PATH@"
+
+    # Top-level build directory
+    # NOTE: This is not especially useful on installed systems, but is useful for
+    # running various programs from a source checkout
+    BUILD_DIR = "@abs_top_builddir@"
```

Comment:
Maybe this one should be `_BUILD_DIR` since nothing external should be running from a checkout

BTW we may as well do the comments as docstrings (even though Python itself doesn't do anything with docstrings for variables/constants). We could either list them all in the class docstring, or I believe sphinx will pick it up if after the constant e.g.

    BASH_PATH = "@BASH_PATH@"
    """ Path to the bash shell """

That also brings up the point that we should use sphinx to auto-generate the module documentation. That's what it was originally designed for, so it should be straightforward. Probably in the pacemaker-doc package.

Reply:
I'm pretty sure the docstring gets ignored when applied to a variable, unless sphinx adds its own custom parsing to detect it.
* https://peps.python.org/pep-0257/#what-is-a-docstring
* https://stackoverflow.com/questions/8820276/docstring-for-variable

Reply:
I deleted my comment after re-reading yours. You already mentioned that Python ignores docstrings for variables/constants. Not sure if Sphinx has special parsing for that...

Reply:
Yep, it does

Reply:
Fixed the comments and renamed the one to _BUILD_DIR.  I'll add a commit on top to add docs building.